projects/plugins/jetpack/_inc/lib/admin-pages/class.jetpack-admin-page.php

```diff
@@ -283,7 +283,7 @@ public static function load_wrapper_styles() {
 	 */
 	public static function wrap_ui( $callback, $args = array() ) {
 		$defaults          = array(
-			'is-wide'  => false,
+			'is-wide'  => true,
```

Comment:
Sergey's approach in https://github.com/Automattic/jetpack/pull/30277/files#diff-77cc6b621a06e9e27a68759973cdb0af136471286592eb40392f46f568f70b04R156 is probably safer...